src/test/java/uk/gov/pay/ledger/transaction/dao/TransactionDaoIT.java

```diff
@@ -45,7 +45,6 @@ public void shouldInsertTransaction() {
         assertThat(retrievedTransaction.getState(), is(transactionEntity.getState()));
         assertThat(retrievedTransaction.getEmail(), is(transactionEntity.getEmail()));
         assertThat(retrievedTransaction.getCardholderName(), is(transactionEntity.getCardholderName()));
-        assertThat(retrievedTransaction.getExternalMetadata(), is(transactionEntity.getExternalMetadata()));
```

Comment:
We could possibly assert `externalMetadata` from `transactiondetails`

Reply:
Didn't expect this to be a blocker for the PR.
Added in below commit.